Sources/RequestKit/JSONPostRouter.swift

```diff
@@ -22,11 +22,11 @@ public extension JSONPostRouter {
         let task = session.uploadTask(with: request, fromData: data) { data, response, error in
             if let response = response as? HTTPURLResponse {
                 if !response.wasSuccessful {
-                    var userInfo = [String: AnyObject]()
-                    if let data = data, let json = try? JSONSerialization.jsonObject(with: data, options: .mutableContainers) as? [String: AnyObject] {
-                        userInfo[RequestKitErrorKey] = json as AnyObject?
+                    var userInfo = [String: Any]()
+                    if let data = data, let json = try? JSONSerialization.jsonObject(with: data, options: .mutableContainers) as? [String: Any] {
+                        userInfo[RequestKitErrorKey] = json as Any?
```

Comment:
I think something's broken here, I am no longer receiving userInfo dicts on my errors.

Reply:
On Linux or iOS?

Reply:
For some reason the platform checks fixed it 👍